migration/iD11/oracle/202303091741_IDEMPIERE-5622.sql

```diff
@@ -0,0 +1,14 @@
+-- IDEMPIERE-5622 - Language in Window and Info Window customization should be filtered by log in languages
+SELECT register_migration_script('202303091741_IDEMPIERE-5622.sql') FROM dual;
+
+SET SQLBLANKLINES ON
+SET DEFINE OFF
+
+-- Mar 9, 2023, 5:41:26 PM CET
+UPDATE AD_Column SET AD_Val_Rule_ID=toRecordId('AD_Val_Rule','534583fc-de63-45fc-b2b6-fcbd21b9a2f6'),Updated=TO_TIMESTAMP('2023-03-09 17:41:26','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_UU='0e32e607-dbc8-47bd-844f-9bc1fc772b39'
```

Comment:
hmm ... this probably a bug in the generate independent id migration script ticket, toRecordId should only be used for non-official id.

Reply:
I need to fix the where clause part (i.e should use where ad_column_id when ad_column_id is official id) but for the ad_val_rule_id, that's something you have to fix, it is not part of official id. you have either to create official id for '534583fc-de63-45fc-b2b6-fcbd21b9a2f6' or include that in the migration script (not sure though that we would start to accept non official id migration script).

Reply:
Thanks! I chose the wrong reference - I will create a new pull request with the requested changes .